src/AspNet.Security.OAuth.Alipay/AlipayCertificationManager.cs

```diff
@@ -0,0 +1,61 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Numerics;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace AspNet.Security.OAuth.Alipay;
```

Comment:
A static object won't work for a scenario where a user configures the Alipay provider twice with different configurations. It's also bad for testability.